### ISSUE TITLE
Missing AudioSet info 

### ISSUE THREAD

**Issue by krysal** [WordPress/openverse-api#726]
## Description

The API is omitting the audio set info in the [audio_detail](https://api.openverse.engineering/v1/#operation/audio_detail) endpoint.

## Reproduction

1. See this audio (and its related ones) that should include an audio set given its [source page](https://www.jamendo.com/track/35663/mahler-s-cat) info. The album is called "Mahler's Cat"
https://api.openverse.engineering/v1/audio/71a2cf71-cdb0-49d6-94d0-d0f757e37799/
2. Response
```json
{
...
"audio_set": null,
...
}
```

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

**Comment by dhruvkb** [comment]
I was not able to replicate this locally on `main`. For one of our sample data items ('Company' by Kyle Cox), the [details endpoint](http://localhost:8000/v1/audio/69bee165-ee45-40fc-b4ad-3f57f0a48bca/) gives the expected audio set information.

Could this be a bug that we didn't ingest the album information?

**Comment by krysal** [comment]
@dhruvkb Yes, it looks like the problem is in production data. The second query I mentioned in WordPress/openverse#1435 also leads me to believe that.

```sql
-- Audio with thumbnail & without audio set
 SELECT COUNT(id) FROM audio
 WHERE thumbnail IS NOT NULL AND audio_set_foreign_identifier IS NULL;
 
+-------+
| count |
|-------|
| 89138 |
+-------+
SELECT 1
Time: 0.179s
```

**Comment by dhruvkb** [comment]
@krysal should we close this issue then? Or maybe update it to describe the investigative nature of the task.

**Comment by krysal** [comment]
@dhruvkb Yes, I think we can close it. This seems a better fit to the catalog, I included it in WordPress/openverse#1537.
